pyproject.toml

```diff
@@ -71,6 +71,7 @@ pypdf = {version = "^3.9", optional = true}
 pillow = {version = "^10.2.0", optional = true}
 mail-parser = {version = "^3.15.0", optional = true}
 filetype = {version = "^1.2", optional = true}
+tavily-python = "^0.5.0"
```

Comment:
lets move this under `torch` and follow the convention of the other imports by adding `optional = true`

Reply:
missed that, thanks